app/src/main/kotlin/com/wire/android/datastore/GlobalDataStore.kt

```diff
@@ -232,4 +233,9 @@ class GlobalDataStore @Inject constructor(@ApplicationContext private val contex
     fun selectedThemeOptionFlow(): Flow<ThemeOption> =
         getStringPreference(APP_THEME_OPTION, ThemeOption.SYSTEM.toString())
             .map { ThemeOption.valueOf(it) }
+
+    fun enterToSendFlow(): Flow<Boolean> = getBooleanPreference(ENTER_TO_SENT, false)
```

Comment:
as far as it's used only as `enterToSendFlow().first()` maybe make sens to change it to  `suspend fun isEnterToSend(): Boolean`